Wizardry/Core/BattleSys/Source/PreBattleCalc.c

```diff
@@ -569,6 +569,11 @@ void PreBattleCalcSkills(struct BattleUnit * attacker, struct BattleUnit * defen
         }
     }
 #endif
+
+#if defined(SID_Charge) && (SID_Charge < MAX_SKILL_NUM)
+        if (SkillTester(unit, SID_Charge))
+            attacker->battleAttack += gActionData.moveCount/2;
```

Comment:
follow the code style of fe6 repo, add space before and after "/"

Reply:
Done